src/include/platform/internal/GenericPlatformManagerImpl_POSIX.h

```diff
@@ -51,6 +51,9 @@ namespace Internal {
 template <class ImplClass>
 class GenericPlatformManagerImpl_POSIX : public GenericPlatformManagerImpl<ImplClass>
 {
+public:
+    using ThreadIdType = pthread_t;
```

Comment:
You don't seem to use it below.

Reply:
Removed, had code that tried to be generic (but could not after all). Thanks for noticing.